pkg/proxy/apis/kubeproxyconfig/v1alpha1/types.go

```diff
@@ -22,65 +22,65 @@ import (
 
 // ClientConnectionConfiguration contains details for constructing a client.
 type ClientConnectionConfiguration struct {
-	// kubeConfigFile is the path to a kubeconfig file.
+	// KubeConfigFile is the path to a kubeconfig file.
```

Comment:
In godoc for external apis, refer to the field by its JSON name (kubeconfig) because this is documentation that is output into our formal human readable documentation, where the Go-field name doesn't make sense.

Too many same mistakes in this PR, unfortunately.

Reply:
Cool! This makes sense to me. It's easy to revert.

Since you are on this line, please notice that `kubeConfigFile` (comment) vs `kubeconfig` (JSON name)

Reply:
and L33: `cps` vs `qps`